peppy/project.py

```diff
@@ -512,6 +498,17 @@ def get_sample(self, sample_name):
 
         return samples[0]
 
+    def activate_subproject(self,subproject):
```

Comment:
This will overwrite existing entries, but `activate_subproject` _feels_ to me like an operation that would be strictly additive rather than mutative. I don't have a strong preference, but what about `use_subproject`?

Reply:
we named it "activate" in `pepr`. But I'm ok with changing it

Reply:
Sounds good, since the intent is to update the existing Project rather than create a new one I think that makes more sense than my suggestion